website/source/docs/agent/options.html.md

```diff
@@ -592,6 +592,15 @@ Consul will not enable TLS for the HTTP API unless the `https` port has been ass
 * <a name="advertise_addr"></a><a href="#advertise_addr">`advertise_addr`</a> Equivalent to
   the [`-advertise` command-line flag](#_advertise).
 
+* <a name="allow_node_renaming"></a><a href="#allow_node_renaming">`allow_node_renaming`</a>
+  Allows to rename node with IDs. Renaming of nodes will be allowed as it does not collides
+  with another node with different ID but same name (case insensitive comparison).
+  It also enforces case insensitive match for node names when performing registration on
+  nodes without IDs.
+  Since it is a breaking change, its value is `false` by default in Consul version 1.2.1+ but
```

Comment:
I'm not sure, IMHO, that this is the right place to state that the default value will/may change.

Reply:
Not sure neither, maintainers will decide